PyRDF/backend/Local.py

```diff
@@ -13,6 +13,41 @@ class Local(Backend):
         backend.
 
     """
+    def __init__(self, config={}):
+        """
+        Creates a new instance of the
+        Local implementation of `Backend`.
+
+        Parameters
+        ----------
+        config (optional)
+            The config object for the required
+            backend. The default value is an
+            empty Python dictionary `{}`.
+
+        """
+        super().__init__(config)
+        self.supported_operations = [
```

Comment:
I would expect the `Local` backend to copy the list of the superclass and remove those items that are not supported.

Reply:
Sure @etejedor . Will do this 👍